### ISSUE TITLE
ft.com

### ISSUE THREAD

**Issue by hobbesd** [reek/anti-adblock-killer#3172]
safari
Adblock
aaklist
tampermonkey
aakscript

**Comment by uBlock-user** [comment]
That's paywall, not anti adblock script.

**Comment by hobbesd** [comment]
Its Paywall AND adblock script. If you just open the homepage (without clicking on any of the articles) it tells you to disable the adblocker because the site needs revenue from both subscribers AND ads. I will add a screenshot when i get the chance.

**Comment by uBlock-user** [comment]
Paywall won't be patched and I can't reproduce any anti adblock script.

**Comment by jspenguin2017** [comment]
I can't reproduce it. 
![image](https://cloud.githubusercontent.com/assets/7283682/25773721/aec24fc6-323f-11e7-9ff4-14e49815a714.png)
I see the pay wall, but nothing else on the home page. If you can dump the source of a real article, we might be able to help. I'm not sure which ToS you accepted, so make sure you won't run into trouble when you dump their source. 
![image](https://cloud.githubusercontent.com/assets/7283682/25773706/68778ffe-323f-11e7-966d-82573128563a.png)

**Comment by uBlock-user** [comment]
@jspenguin2017 Do we patch paywalls ?

**Comment by jspenguin2017** [comment]
@uBlock-user No, and that's impossible (if the site master did it right). We do patch "disable Adblock or pay" type of paywall.

**Comment by uBlock-user** [comment]
I blocked everything from CSS to scripts, XHR, frame, other, media, still no change. It's like embedded inside the document and not something you can disable or remove.

**Comment by jspenguin2017** [comment]
@uBlock-user It's done on server side, if the server refuse to serve you the content, you can't get it.

**Comment by uBlock-user** [comment]
Yep, that's it, that appears to be the case.

**Comment by jspenguin2017** [comment]
> because the site needs revenue from both subscribers AND ads

Well, they are getting neither. 

@uBlock-user Yea. Agreed.

**Comment by uBlock-user** [comment]
@hobbesd If you're desperate to read articles from the website anyhow, you may try this - https://www.ghacks.net/2016/02/26/read-articles-behind-paywalls-by-masquerading-as-googlebot/

**Comment by jspenguin2017** [comment]
@uBlock-user Looks like pay walled articles are not indexed by Google. I don't think that would work.

**Comment by jspenguin2017** [comment]
@uBlock-user Google is able to index those pages. And if you click into the page through Google, you'll be HTTP302'ed to the real content page... Looks like the site master did it wrong. You need to lock Google's IP, not just check the UserAgent.

**Comment by uBlock-user** [comment]
@jspenguin2017 so switching to Google's IP works ?

**Comment by uBlock-user** [comment]
Take for example this one - https://www.ft.com/content/cd19af44-3287-11e7-99bd-13beb0903fa3

You can read it via Google's cached copy - https://webcache.googleusercontent.com/search?q=cache:A8ScaApO1h8J:https://www.ft.com/content/cd19af44-3287-11e7-99bd-13beb0903fa3+&cd=1&hl=en&ct=clnk

**Comment by jspenguin2017** [comment]
@uBlock-user You can't take Google's IP, but yes, you can definitely use Google's Webcache. The web page can, however, instruct Google to not cache their page.

**Comment by hobbesd** [comment]
its not the paywall I was complaining about. Here is an article if you don't believe me:
https://digiday.com/media/financial-times-toughens-stance/

In any case, the anti Adblock script seems to be gone now (or they disabled it), because I don't see it anymore when I refresh the page.

However, I did dump the page source when I last saw it. I attached it to this post. maybe you can find it in there.
[Financial Times.zip](https://github.com/reek/anti-adblock-killer/files/981701/Financial.Times.zip)

**Comment by jspenguin2017** [comment]
@hobbesd Well, it's gone then it's gone, we aren't going to fix something that is gone. If it comes back, it might be different.

**Comment by hobbesd** [comment]
If you read the article, it says that it only shows the message to users who have registered an account but not provided any payment details. Since I have provided payment details, this doesn't concern me anymore. But you might still want do do something about it.

**Comment by jspenguin2017** [comment]
@hobbesd It's probably hidden in one of the scripts that are loaded on the fly, without a live sample, I can't debug it.

**Comment by uBlock-user** [comment]
@jspenguin2017 Yes we can in sort of a way by installing Google's Data Saver extension and changing the UA to Googlebot.

@hobbesd I cannot reproduce any anti adblock script in action, so can't offer you any solutions if you do. Why did you register ?

**Comment by dnivi3** [comment]
I am having this issue, see screenshot below. 
![screen shot 2017-12-14 at 14 20 32](https://user-images.githubusercontent.com/1163576/33994253-0adac894-e0da-11e7-9e00-ee3f417a0595.png)

Page source is here: 
[Financial Times_14-12-17.zip](https://github.com/reek/anti-adblock-killer/files/1559335/Financial.Times_14-12-17.zip)

**System information:**
macOS 10.12.6 (16G1114)
Chromium Version 63.0.3239.84 (Developer Build) (64-bit)
uBlock Origin 1.14.20
uBlock Origin Extra 2.58
Tampermonkey 4.4
HTTPS Everywhere 2017.12.6

Also, sorry for necro'ing this thread.

**Comment by dnivi3** [comment]
Update: I found out how to block the script that makes that popup show. The script is the following:


www.ft.com https://www.ft.com/__assets/creatives/ad-blocking/sourcepoint-script-19-10-2017.js script block | script | https://www.ft.com/__assets/creatives/ad-blocking/sourcepoint-script-19-10-2017.js
| -- | -- | --
